.github/workflows/test.yml

```
@@ -0,0 +1,23 @@
+name: Kubernetes Javascript Client - Validation
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '10', '8' ]
+    name: Node ${{ matrix.node }} validation
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - run: npm install
```

Maybe we can add back 
https://github.com/kubernetes-client/javascript/blob/master/.travis.yml#L6-L9

```suggestion
      # Pre check to check that version matches package-lock.json
      # needs to happen before npm install
      - run: node version-check.js
      - run: npm install
```

---

done.